### PR TITLE
Ifedayo/futr 1043 error when building dane with mastra dev

### PR DESCRIPTION
**Pull request by adeniyii**

### CODE REVIEW

**Review by taofeeq-deru — APPROVED** [review]

**Review by abhiaiyer91 — APPROVED** [review]

File: packages/cli/src/commands/dev.ts [hunk 1]
```diff
@@ -107,16 +108,28 @@ export async function dev({
...
 
-  const proc = execa('node', ['server.mjs'], {
-    cwd: dotMastraPath,
-    env: {
-      port: `${port} || 4111`,
-      MASTRA_TOOLS_PATH,
-    },
-  });
+  try {
+    console.log('Starting server...');
```
Comment (TheIsrael1) [inline packages/cli/src/commands/dev.ts:112]: we can remove the stray logs now i think right?

File: packages/cli/src/utils/bundle.ts [hunk 2]
```diff
@@ -166,13 +173,17 @@ export async function bundle(dirPath: string, options?: { outfile?: string }) {
         'wasi',
         'worker_threads',
         'zlib',
+        'chromium-bidi/lib/cjs/bidiMapper/BidiMapper',
```
Comment (TheIsrael1) [inline packages/cli/src/utils/bundle.ts:176]: do we still need these additions

File: packages/cli/src/commands/dev.ts (hunk 1 above)
Comment (adeniyii) [reply packages/cli/src/commands/dev.ts:112]: Not sure, I think it's useful for the user to know when the server starts

File: packages/cli/src/utils/bundle.ts (hunk 2 above)
Comment (adeniyii) [reply packages/cli/src/utils/bundle.ts:176]: yeah we still need this
this is accounted for by `@mastra/rag`

**Pull request merged**